modules/web/src/app/settings/admin/defaults/template.html

```diff
@@ -59,6 +59,8 @@
             <span>OPA Integration</span>
             <div class="km-icon-info km-pointer"
                  matTooltip='Set "OPA Integration" checkbox on cluster creation to enabled by default. Enable "Enforce" to make users unable to edit the checkbox.'></div>
+            <i class="km-icon-warning km-pointer"
+               matTooltip="OPA (Open Policy Agent) integration is deprecated. Kyverno will replace it as an EE-only feature for policy management."></i>
```

Comment:
/hold

Please explicitly mention the release number in which this was deprecated and when this is scheduled for removal. Why did we even change it from the last message? :D